tests/everflow/test_everflow_ipv6_egress.py

```diff
@@ -0,0 +1,706 @@
+            tcp_sport=tcp_sport,
+            tcp_dport=tcp_dport,
+            tcp_flags=tcp_flags
+        )
+        
+        if vlan_id:
+            pkt = testutils.simple_tcpv6_packet(
+                eth_dst=dl_dst,
+                eth_src=ptfadapter.dataplane.get_mac(0, 0),
+                ipv6_src=src_ip,
+                ipv6_dst=dst_ip,
+                ipv6_dscp=dscp,
+                tcp_sport=tcp_sport,
+                tcp_dport=tcp_dport,
+                tcp_flags=tcp_flags,
+                dl_vlan_enable=True,
+                vlan_vid=vlan_id
+            )
+        
+        return pkt
+    
+    def _base_udpv6_packet(
+        self,
+        ptfadapter,
+        setup,
+        direction=DOWN_STREAM,
+        src_ip=None,
+        dst_ip=None,
+        udp_sport=0x1234,
+        udp_dport=0x50,
+        dscp=8
+    ):
+        """
+        Generate a base UDPv6 packet for testing.
+        """
+        if src_ip is None:
+            src_ip = self.DEFAULT_SRC_IP
+        if dst_ip is None:
+            dst_ip = self.DEFAULT_DST_IP
+        
+        return testutils.simple_udpv6_packet(
+            eth_dst=setup[direction]["ingress_router_mac"],
+            eth_src=ptfadapter.dataplane.get_mac(0, 0),
+            ipv6_src=src_ip,
+            ipv6_dst=dst_ip,
+            ipv6_dscp=dscp,
+            udp_sport=udp_sport,
+            udp_dport=udp_dport
+        )
+    
+    def _base_icmpv6_packet(
+        self,
+        ptfadapter,
+        setup,
+        direction=DOWN_STREAM,
+        src_ip=None,
+        dst_ip=None,
+        icmp_type=128,
+        icmp_code=0,
+        dscp=8
+    ):
+        """
+        Generate a base ICMPv6 packet for testing.
+        """
+        if src_ip is None:
+            src_ip = self.DEFAULT_SRC_IP
+        if dst_ip is None:
+            dst_ip = self.DEFAULT_DST_IP
+        
+        return testutils.simple_icmpv6_packet(
+            eth_dst=setup[direction]["ingress_router_mac"],
+            eth_src=ptfadapter.dataplane.get_mac(0, 0),
+            ipv6_src=src_ip,
+            ipv6_dst=dst_ip,
+            icmp_type=icmp_type,
+            icmp_code=icmp_code
+        )
+    
+    def _get_tx_port_id_list(self, tx_ports):
+        """
+        Extract list of transmit port IDs from port configuration.
+        """
+        return BaseEverflowTest._get_tx_port_id_list(tx_ports)
+
+
+class TestEverflowV6EgressAclEgressMirror(EverflowIPv6EgressTests):
+    """
+    Test class for IPv6 Egress ACL with Egress Mirror.
+    
+    Tests SAI_ACL_TABLE_ATTR_FIELD_* attributes for IPv6:
+    - OUTER_VLAN_ID
+    - ACL_IP_TYPE
+    - SRC_IPV6
+    - DST_IPV6
+    - ICMPV6_CODE
+    - ICMPV6_TYPE
+    - IPV6_NEXT_HEADER
+    - L4_SRC_PORT
+    - L4_DST_PORT
+    - TCP_FLAGS
+    - DSCP
+    - L4_SRC_PORT_RANGE
+    - L4_DST_PORT_RANGE
+    """
+    
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_acl_table(self, setup_info, setup_mirror_session, config_method):
```

Comment:
Since ACL rule is also added in this fixture, probably it's better to update the function name accordingly?